lib/modules/platform/github/readme.md

```diff
@@ -25,6 +27,26 @@ You can choose where you want to set `endpoint`:
     If you're self-hosting Renovate on GitHub.com with GitHub Actions in forking mode, and want Renovate to apply labels then you must give the PAT `triage` level rights on `issues`.
     The `triage` level allows the PAT to apply/dismiss existing labels.
 
+## Running using a Fine-grained Token
```

Comment:
```suggestion
## Running using a fine-grained Token
```

I think we can lowercase this?

Reply:
I'd be inclined to lowercase `Token` as well if that's desired.

Whether we can/can't is too complicated for me which is why I didn't respond earlier.

Reply:
```suggestion
## Running using a fine-grained token
```

This is a _new_ heading, so it should be safe to change the capitalization. 😄